hevc/hevcdecoderconfigurationrecord_test.go

```diff
@@ -36,15 +39,18 @@ func TestCreateDecConfRec(t *testing.T) {
 		vpsNalus := createNalusFromHex(tc.vpsHex)
 		spsNalus := createNalusFromHex(tc.spsHex)
 		ppsNalus := createNalusFromHex(tc.ppsHex)
+		seiNalus := createNalusFromHex(tc.seiHex)
 		dcr, err := CreateHEVCDecConfRec(vpsNalus, spsNalus, ppsNalus,
 			tc.complete, tc.complete, tc.complete, tc.includePS)
+		seiArray := *NewNaluArray(tc.complete, NALU_SEI_PREFIX, seiNalus)
```

Comment:
This is not a new thing, but I think it is worth changing the `NewNaluArray` to return `NaluArray` and not a pointer.  If you do it, do it in a separate commit so that it is visible that there is a change.

Reply:
I agree on this point. I updated it on a separated commit